<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenposeGUI</class>
 <widget class="QMainWindow" name="OpenposeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenposeGUi</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_camera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/camera.png</normaloff>
     <normalon>../icon/stop.png</normalon>../icon/camera.png</iconset>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>../icon/save.png</normaloff>../icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_autosave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/auto.png</normaloff>../icon/auto.png</iconset>
   </property>
   <property name="text">
    <string>自动保存</string>
   </property>
  </action>
  <action name="action_setting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/setting.png</normaloff>../icon/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_filetree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/dir.png</normaloff>../icon/dir.png</iconset>
   </property>
   <property name="text">
    <string>目录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
